plugins/kluster-capacity.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kluster-capacity
+spec:
+  version: v0.1.0
+  homepage: https://github.com/k-cloud-labs/kluster-capacity
```

Comment:
Please import auth plugins https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/#auth-plugins otherwise it won't work on GCP clusters etc.